### PR TITLE
Unihan phase 1

### PR DESCRIPTION
**Pull request by duncdrum**
This builds on #1896 and is ready for feedback and discussion, I ll remove the WIP, once phase 0 has been reviewed and merged.
As it stands both `@value` and `@name` are required on the three new elements.  
Since automation of definition generation is for a later phase i have made `@version` recommended, with a semi-open list.
My main question is how to go about the deprecation and in how far i should switch out the old prose and examples in the documentation already? But deprecation stuff is phase 2. 
The idea is that phase 1 could be included in a release, without folks really noticing much about it. i think this pretty much achieves that.

see #1805

### CODE REVIEW

**Comment by duncdrum** [comment]
i can run `make test` locally but travis catches an actual problem,  I'm not sure where to add the reference to the new attribute class i just created. Also if any css wizard has some suggestions for how to get the eqivalent of `overlow: scroll` working for the spec pages without introducing new divs it would be much appreciated.

**Comment by duncdrum** [comment]
could someone point me to the documentation for how i can grab a new detest log to replace the one in expected result?

**Comment by martindholmes** [comment]
What we usually do for the detest log is to diff the one generated in the build process against the one in expected-results; if the only changes we see are those we expect from the code change, and the changes are all correct, then we copy the generated one over the expected-results one and commit.

It's crude, but we haven't been able to come up with a better approach. The important thing is to pay close attention when doing the diff, in case something unexpected sneaks past.

**Comment by duncdrum** [comment]
what command do you use to make the log stay around after diffing? `make test` removes the generated log immediately so i can't check it.

**Comment by martindholmes** [comment]
I thought it was only deleted if the test completed successfully. Are you running locally, or just looking at Travis?

**Comment by duncdrum** [comment]
locally using docker
```
Validate ODD file using RELAX NG schema
rnv detest.rnc detest.xml > detest2.log 2>&1
Makefile:93: recipe for target 'detest' failed
make[1]: [detest] Error 1 (ignored)
Validate XML file using Schematron
java -Xmx752m -jar ../Utilities/lib/saxon9he.jar -ext:on detest.xml detest.xsl > detest3.log 2>&1
grep "\[xslt\]" detest1.log | sed 's/Processing.*detest.odd//' | sed 's/Loading.*message.isosch.xsl//' > detest.log
sed 's/.*detest.xml:/detest.xml:/' detest2.log >> detest.log
sed 's/.*detest.xml:/detest.xml:/' detest3.log >> detest.log
diff detest.log expected-results/detest.log
rm -f detest*log
```

travis does catch something, with a slighly more meaningful error message, but i also want to add things to detest...

**Comment by duncdrum** [comment]
i think the command should be `rm -f detest*.log`?

**Comment by martindholmes** [comment]
That line has been unchanged for six years, and it was Sebastian who last touched it, so I wouldn't quickly assume that it's wrong. :-)

**Comment by duncdrum** [comment]
yeah red herring, i changed it to
```
	rm -f detest1.log detest2.log detest3.log
```
so i could keep my detest.log, but when i compare the newly generated one with the one in  in expected results i see two identical files, Yet:
`Makefile:93: recipe for target 'detest' failed`

File: P5/Source/Guidelines/en/WD-NonStandardCharacters.xml [hunk 1]
```diff
@@ -1356,11 +1356,16 @@ The selection and combination of modules to form a TEI schema is described in
 <include xmlns="http://www.w3.org/2001/XInclude" href="../../Specs/charName.xml"/>
 <include xmlns="http://www.w3.org/2001/XInclude" href="../../Specs/charProp.xml"/>
 <include xmlns="http://www.w3.org/2001/XInclude" href="../../Specs/charDecl.xml"/>
+<!-- duncdrum: Why should / can I not include the att class here??  -->
+<!-- <include xmlns="http://www.w3.org/2001/XInclude" href="../../Specs/att.gaijiProp.xml"/> -->
```
Comment (duncdrum) [inline P5/Source/Guidelines/en/WD-NonStandardCharacters.xml]: If I include this here i get two name and two value properties on the elements which leads odd validation errors (two name attribtes) and to display issues with the html spec page.

**Comment by duncdrum** [comment]
now that i can reproduce the CI failure i m going to clean this up

**Pull request closed without merging**